HousingRepairsSchedulingApi.Tests/UseCasesTests/BookAppointmentUseCaseTests.cs

```diff
@@ -83,7 +83,7 @@ public BookAppointmentUseCaseTests()
         [Fact]
 #pragma warning disable xUnit1026
 #pragma warning disable CA1707
-        public async void GivenAnInvalidOrderComments_WhenExecute_ThenExceptionIsThrown()
+        public async void GivenOrderCommentsLongerThan255Characters_WhenExecute_ThenExceptionIsThrown()
 #pragma warning restore xUnit1026
```

Comment:
This `CA1707` pragma is disabled on [line 85](https://github.com/Newarkandsherwood/HousingRepairsSchedulingApi/pull/17/files/6eda9533e96e0b0b691b867a42921366f22f08c0..9970d11c3f0ef19d7b14061158769babb1e55015#diff-ee46ab0b3c32b692f074acb3609948491b12173ca859dbac5b06124a5f31526dR85) and should be re-enabled appropriately (i.e. after the method signature)
```suggestion
#pragma warning restore CA1707
#pragma warning restore xUnit1026
```